website-v2/src/routes/+page.svelte

```diff
@@ -1,133 +1,176 @@
 <script>
+    import { ArrowRight, LibraryBig, Microscope, Play, Rocket } from 'lucide-svelte';
+
     import AnimatedLogo from '$lib/animated-logo.svelte';
-    import AdministrationSVG from '$lib/assets/administration.svg';
-    import CodeTypingSVG from '$lib/assets/code-typing.svg';
-    import SpreadsheetsSVG from '$lib/assets/spreadsheets.svg';
+    import PublicodesSchemaSVG from '$lib/assets/publicodes-schema.svg';
     import PublicodesEditor from '$lib/publicodes/editor.svelte';
     import Button from '$lib/ui/button.svelte';
-    import Card from '$lib/ui/card.svelte';
+
+    const microscope = $derived(Microscope);
+    const iconSize = 36;
+    const iconStrokeWidth = 1.5;
 </script>
 
-<header class="not-prose overflow-hidden bg-gradient-to-r from-primary-700 to-slate-400 py-4">
-    <div class="mx-auto gap-12 p-6 lg:container sm:flex md:p-10 lg:flex-row-reverse">
-        <div class="flex justify-center self-center max-sm:scale-75 xl:flex-1 xl:scale-125">
+<header class="not-prose w-full overflow-hidden bg-primary-50">
```

Comment:
Visuellement j'aime un peu mieux le bandeau contrasté actuel

<table>
<tr><td>Actuel</td><td>PR</td></tr>
<tr><td>

![image](https://github.com/publicodes/publicodes/assets/1730702/f550fac0-aead-40c8-a6e9-5261ad400572)

</td><td>

![image](https://github.com/publicodes/publicodes/assets/1730702/241fa161-9cae-4347-802d-3fcd1af11326)

</td></tr>
</table>

Reply:
Je crois que pour ma part, je préfère la nouvelle version. Je préfère le côté épuré, et cela met davantage en avant le logo. Le dégradé vers le gris m'a lassé à force d'itérer sur la page je pense :)

Reply:
Par contre un peu moins fan des sections en couleur `administration / expert / dev` qui font un peu austère. On peut itérer aussi dessus.